guides/user/pipeline/triggers/gcs/index.md

```diff
@@ -10,18 +10,18 @@ redirect_from: /guides/user/triggers/gcs/
 
 This guide explains how to configure Spinnaker to trigger pipelines based on
 changes in a [Google Cloud Storage](https://cloud.google.com/storage/) (GCS)
-bucket, and inject changed GCS objects as [artifacts](/reference/artifacts)
+bucket and inject changed GCS objects as [artifacts](/reference/artifacts)
 into a pipeline.
 
 This functionality uses Google's
 [Pub/Sub](https://cloud.google.com/pubsub/docs/overview) system for delivering
-messages to Spinnaker, and must be configured to send messages to
+messages to Spinnaker and must be configured to send messages to
 Spinnaker's event bus as shown below.
 
 # Prerequisite configuration/setup
 
-If you (or your Spinnaker admin) has already configured Spinnaker to listen to
-a Pub/Sub messages from the GCS bucket you plan to publish objects to, you can
+If you (or your Spinnaker admin) already configured Spinnaker to listen to
+a Pub/Sub messages from the GCS bucket and you plan to publish objects to, you can
```

Comment:
I think this one was correct in the original. The sentence could also be written "If you...already configured Spinnaker to listen to Pub/Sub message from the GCS bucket _that_ you plan to publish objects to..." But we can omit the "that."

(And oops, I see there's a superfluous "a" in there, before "Pub/Sub".)